src/hotspot/share/opto/superword.hpp

```diff
@@ -502,11 +494,6 @@ class SuperWord : public ResourceObj {
 
 #ifndef PRODUCT
   // TraceAutoVectorization and TraceSuperWord
-  bool is_trace_superword_alignment() const {
-    // Too verbose for TraceSuperWord
-    return _vloop.vtrace().is_trace(TraceAutoVectorizationTag::SW_ALIGNMENT);
-  }
```

Comment:
Note: All the old verbose tracing is now removed. I now only use `is_trace_superword_adjacent_memops`.